README.md

```diff
@@ -1637,122 +1634,147 @@ func ==(lhs: Item, rhs: Item) -> Bool {
     return lhs.GUID == rhs.GUID
 }
 
+enum Result<T> {
+	case value(T)
+	case failure(Error)
+
+	func success(completionHandler: (T) -> ()) {
+		switch self {
```

Comment:
Could you change the indentation in these snippets from tabs to 4 spaces in this snippet?